sbin/wifibox

```diff
@@ -734,8 +740,22 @@ vm_manager() {
 
     log debug "Nice priority: ${_nice_priority}"
     log debug "Arguments: ${_bhyve_args}"
-    ${NICE} -n ${_nice_priority} \
-	    ${BHYVE} ${_bhyve_args} 2>&1 | capture_output debug bhyve
+
+    _kill_child() {
+	_jobs="$(jobs -p)"
+
+	[ -n "$_jobs" ] && ${KILL} -SIGTERM $_jobs
+	exit 0
+    }
+
+    trap _kill_child SIGTERM
+
+    { ${NICE} -n ${_nice_priority} \
+	    ${BHYVE} ${_bhyve_args} 2>&1 | capture_output debug bhyve; } &
+    wait
```

Comment:
We will have to explicitly `wait` here for the forked process (`$!`) since that is what matters here.